test/fennec/auth_test.exs

```diff
@@ -0,0 +1,303 @@
+defmodule Fennec.AuthTest do
+  use ExUnit.Case
+
+  alias Jerboa.Params
+  alias Jerboa.Format
+  alias Jerboa.Format.Body.Attribute.{Username, ErrorCode,
+                                      RequestedTransport, Nonce, Realm}
+
+  @recv_timeout 5000
+
+  setup ctx do
+    Application.put_env(:fennec, :secret, "abc")
+    test_case_id = ctx.line
```

Comment:
test context. Initial value of this variable is described [here](https://hexdocs.pm/ex_unit/ExUnit.Callbacks.html) (section "Context").